docs/develop/python/message-passing.mdx

```diff
@@ -187,17 +211,19 @@ class WorkflowB:
         handle = workflow.get_external_workflow_handle_for(WorkflowA.run, "workflow-a")
         await handle.signal(WorkflowA.your_signal, "signal argument")
 ```
+When an External Signal is sent:
+- A [SignalExternalWorkflowExecutionInitiated](/references/events#signalexternalworkflowexecutioninitiated) Event appears in the sender's Event History.
+- A [WorkflowExecutionSignaled](/references/events#workflowexecutionsignaled) Event appears in the recipient's Event History.
 
-### Signal-With-Start {#signal-with-start}
 
-**How to send a Signal-With-Start using the Python SDK.**
 
-Signal-With-Start is used from the Client.
-It takes a Workflow Id, Workflow arguments, a Signal name, and Signal arguments.
+#### Signal-With-Start {#signal-with-start}
 
-If there's a Workflow running with the given Workflow Id, it will be signaled. If there isn't, a new Workflow will be started and immediately signaled.
+Signal-With-Start is sent by a Client.
+To send a Signal-With-Start, use the [`start_workflow()`](https://python.temporal.io/temporalio.client.Client.html#start_workflow) method and pass the `start_signal` argument with the name of your Signal.
+
+If there's a Workflow running with the given Workflow Id, the Signal will be sent to it. If there isn't, a new Workflow will be started and the Signal will be sent immediately on start.
```

Comment:
Good call, added this to the `Waiting` section:

> This is necessary if your Signal or Update handlers require something in the main Workflow method to be done first, since a Signal or Update handler can execute before the main Workflow method has started.